hpaction/build_hpactionvars.py

```diff
@@ -192,6 +192,7 @@ def user_to_hpactionvars(user: JustfixUser) -> hp.HPActionVariables:
     ]
     v.sue_for_repairs_tf = True
     v.request_fee_waiver_tf = True
+    v.sue_for_harassment_tf = False
```

Comment:
Heh, the PDF wasn't showing the address of the management company at all until I explicitly added this line. So finicky.